riff-raff.yaml

```diff
@@ -6,7 +6,6 @@ deployments:
   cloudformation:
     type: cloud-formation
     app: atom-preview
-    parameters:
```

Comment:
CI is just not working - I've changed [d3b5b4c](https://github.com/guardian/atom-preview/pull/28/commits/d3b5b4c614357353af51aea7f115c44fdd70a773) to copy [eb18bad](https://github.com/guardian/atom-preview/pull/28/commits/eb18bad56e2535b9295d9bff6c478351f3150149) (when the CI was passing) to no avail - should we close this PR, @jonathonherbert?

Reply:
I've just run CI in main, and it looks like it's broken there too – it may be that something in npm has changed underneath us.

This PR is still worth keeping, but I think this YAML file is incorrectly formatted, because line 9 has been removed, possibly accidentally. You can test this by pasting the YAML into riff-raff's [validation tool](https://riffraff.gutools.co.uk/configuration/validation) – it's currently invalid, but re-adding `parameters:` fixes.

Happy to pair to help fix the CI! Also happy to approve/merge this PR once the YAML is corrected, regardless of state of CI, as it's very simple and I think obviously correct.